.github/CONTRIBUTING.md

```diff
@@ -64,6 +62,72 @@ To build containerized `moby/buildkit:local` and `moby/buildkit:local-rootless`
 make images
 ```
 
+### Run BuildKit
+
+You can launch the backend BuildKit daemon either directly or in a container:
+
+```bash
+# run the daemon directly
+$ sudo buildkitd --addr unix:///run/buildkit/buildkitd.sock
+$ export BUILDKIT_HOST=unix:///run/buildkit/buildkitd.sock
+$ sudo buildctl debug info
+BuildKit: github.com/moby/buildkit v0.11.0-rc3-506-g539bab193.m 539bab193c28d3ce731e6013f471ba24848f5c41.m
+```
+
+```bash
+# run the daemon in a container
+$ docker run --rm -d --name buildkitd --privileged moby/buildkit:local
+$ export BUILDKIT_HOST=docker-container://buildkitd
+$ buildctl debug info
+BuildKit: github.com/moby/buildkit v0.11.0-rc3-623-g2ff0d2a2f.m 2ff0d2a2f53663aae917980fa27eada7950ff69c.m
+```
```

Comment:
Wondering; could we have a docker-compose.yml that does all this? (build the image, starts it with the right options?)